src/utils/read-files.js

```diff
@@ -0,0 +1,18 @@
+// Get array of files ([File, File]), open each asynchronously
+// and return array of strings that should be passed as src:
+// <img src={images}>
+
+export default async (arr) => {
+  let urls = [];
+  for (let i = 0; i < arr.length; i++) urls.push(await readFileAsync(arr[i]));
```

Comment:
This way, it waits until the current file is completely processed to move on to the next. Better use `Promise.all` with `map`
```suggestion
  urls = await Promise.all(arr.map(readFileAsync));
```
Basically, the whole function can be written in one line:
```js
export default (arr) => Promise.all(arr.map(readFileAsync));
```

Reply:
Oh, thanks. That's really good